packages/terra-progress-bar/tests/wdio/progress-bar-spec.js

```diff
@@ -0,0 +1,7 @@
+/* global browser, Terra */
+
+describe('Progress Bar Gradient', () => {
+  beforeEach(() => browser.url('/#/raw/tests/progress-bar/progress-bar-gradient'));
+  Terra.should.beAccessible();
```

Comment:
Can there be an empty line before this accessibility code so that the test statements are separated from the beforeEach hook which groups test statements together and also makes the test more readable?

Reply:
Sure. Added an empty line. Thanks :)